core/templates/components/hero/hero.twig

```diff
@@ -0,0 +1,57 @@
+{#
+/**
+ * @file
+ * Hero Component
+ *
+ * A big graphic for use as a focal point or a large video.
+ *
+ * Available variants:
+ *   - .su-hero--youtube: youtube iframe embed option
+ *
+ * Available variables:
+ * - attributes: For additional HTML attributes not already provided.
+ * - modifier_class: Additional css classes to change look and behaviour.
+ * - hero_media: iframe or HTML5 video players
+ * - hero_image: A full HTML entity for an image. Can be img or picture.
+ * - hero_super_headline: Text heading
+ * - hero_headline: Text heading
+ * - hero_body: An open variable for anything. Generally this would be for
+ *     WYSIWYG editor contents.
+ * - hero_cta_link: The href value for the cta field link
+ * - hero_cta_attributes: Any html attributes on the cta link that need to be
+ *     added. For example: _target.
+ * - hero_cta_label: The clickable text for the CTA link
+ * - hero_button_link: The href property of the button component.
+ * - hero_button_label: The text label of the button component.
+ */
+#}
+<div{{ attributes }} class="su-hero {{ modifier_class }}">
+
+  {# Hero Image or Video #}
+  <div class="su-hero__media">
+    {% if 'su-hero--youtube' in modifier_class %}
```

Comment:
This is my trick to get the KSS styleguide to show both img and youtube video. KSS does not support multiple data sets for the modifier_classes. Maybe one day it will.

Reply:
Nice trick!